__tests__/app.test.js

```diff
@@ -354,6 +356,28 @@ describe("DELETE api/comments/:comment_id", () => {
   });
 });
 
+describe("GET /api/users", () => {
+  test("Status 200: responds with an array containing all users", () => {
+    return request(app)
+      .get("/api/users")
+      .expect(200)
+      .then(({ body: { users } }) => {
+        expect(users[0]).toMatchObject({
```

Comment:
iterating over and checking each object looks a certain way would be best here, we're only sure what the first one looks like in this case.